mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
This is not a really fair comment. Nothing in MNE works with the sklearn API. We don't do `time_frequency.fit_transform()`, `foward.fit_transform()` or `inverse.fit()`. Users want to know what is being done at each time point, i.e. what is the classifiers, not how the parallel is dealt. This is dev side, and optimization issues.

I'd be more than happy if you fully redesign the GAT in a clearer way while getting equal or better optimization and functions. But personally I don't consider this a priority for MNE, which is still lacking the basic tutorial, stats and across-subjects functionalities.

I also think this discussion is a bit repeating itself now, and this PR isn't really the best place for it.